ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -130,6 +130,8 @@ def get_definitions(base):
     if scheduler in ['slurm']:
        strings.append(f'''\t<!ENTITY PARTITION_SERVICE "{base['QUEUE_SERVICE']}">\n''')
     strings.append(f'\t<!ENTITY SCHEDULER  "{scheduler}">\n')
+    if machine in ['WCOSS2']:
+       strings.append(f'\t<!ENTITY NODESIZE  "128">\n')
```

Comment:
Should this be based on the nodesize set above?

Reply:
It should...this is just hardcoded for ease/time. See comment below in PR.

Reply:
What is the else value for this `NODESIZE`.
If some one works with this branch on orion or any other, this will fail to put a `NODESIZE` value.  It is required now in the task.

Reply:
Can one of you guys help finish adding NODESIZE to the setup scripts then? I don't think I'm going to have time before hand-off. If it's not remedied before hand-off then it can be sorted out in the feature/ops-orion branch when I update that (that's where we currently support ops on the R&Ds).

Reply:
Incorporated @aerorahul patch, thanks! That gets us most of the way there and removes the WCOSS2 hardcoding. We will just need to get it reading from the env files later.